pkg/cvo/cvo.go

```diff
@@ -130,124 +163,193 @@ func (optr *Operator) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer optr.queue.ShutDown()
 
-	glog.Info("Starting ClusterVersionOperator")
+	glog.Infof("Starting ClusterVersionOperator with minimum reconcile period %s", optr.minimumUpdateCheckInterval)
```

Comment:
nit: `glog.Infof("Starting ClusterVersionOperator with minimum reconcile period %s", optr.minimumUpdateCheckInterval)` makes it seem like that is the sync for all actions.

Reply:
That was kind of my intent - what part isn’t reconciled?